src/Stratis.Bitcoin/Builder/FullNodeBuilder.cs

```diff
@@ -53,6 +53,9 @@ public class FullNodeBuilder : IFullNodeBuilder
         /// <inheritdoc />
         public Network Network { get; set; }
 
+        /// <inheritdoc />
+        public IServiceProvider ServiceProvider { get; private set; }
```

Comment:
This can be made private again as SC rules will be use the parent to get access to interfaces